### PR TITLE
Added in appendExclamation to jspm config for flexible support for JS…

### PR DESCRIPTION
**Pull request by erictsai6**
… and TS file types

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "karma-jspm",
-  "version": "2.1.0",
+  "version": "2.2.0",
```
Comment (maxwellpeterson-wf) [inline package.json]: No need to bump the version, we will handle that with our release process.

**Comment by maxwellpeterson-wf** [comment]
One comment otherwise this looks good. Thanks for the contribution!

**Comment by rmconsole-wf** [comment]
@erictsai6 This pull request has merge conflicts, please resolve.

**Comment by evanweible-wf** [comment]
@erictsai6 could you revert the change to the package.json version and merge the latest master?

File: package.json (hunk 1 above)
Comment (erictsai6) [inline package.json]: Sorry about that, i'll change it back

**Comment by erictsai6** [comment]
I've removed the package.json change.  Sorry about that

**Pull request closed without merging**